addons/mail/models/mail_channel.py

```diff
@@ -256,6 +256,16 @@ def write(self, vals):
                 ('res_id', 'in', self.ids)
             ])._moderate_accept()
 
+        # Reflect changes when switching a channel to a mailing one or vice versa
+        if 'email_send' in vals:
+            notifications = []
+            for channel in self:
+                notifications.append([(self._cr.dbname, 'mail.channel', channel.id), {
+                    'info': 'mass_mailing',
+                    'mass_mailing': channel.email_send,
```

Comment:
Not your fault but this is really confusing, why did we even call it `mass_mailing` in JS?
Since this is master I think this would be worth making a preliminary commit to clean this mess and rename it to `email_send` everywhere in JS too.